src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -252,14 +292,22 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }, () => this.canvas.current.applyTag(tag.name));
     }
 
-    /**
-     * Listens for {number key} and calls `onTagClicked` with tag corresponding to that number
-     * @param event KeyDown event
-     */
-    private handleTagHotKey = (event: KeyboardEvent): void => {
-        const key = parseInt(event.key, 10);
+    private onTagRenamed = (oldTag: string, newTag: string) => {
+        const { assets } = this.props.project;
+    }
+
+    private onTagDeleted = (tag: ITag) => {
+        const { assets } = this.props.project;
+    }
+
+    private getTagFromKeyboardEvent = (event: KeyboardEvent): ITag => {
+        let key = parseInt(event.key, 10);
         if (isNaN(key)) {
-            return;
+            try {
+                key = parseInt(event.key.split("+")[1], 10);
```

Comment:
What condition are you checking for here?

Reply:
There is now a keybinding for `Ctrl+{number}` to lock or unlock a tag